packages/fast-components-react-base/src/number-field/number-field.props.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import {
+    ManagedClasses,
+    NumberFieldClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface NumberFieldManagedClasses
+    extends ManagedClasses<NumberFieldClassNameContract> {}
+export interface NumberFieldUnhandledProps
+    extends React.HTMLAttributes<HTMLInputElement> {}

```

Comment:
@nalogiudice we should remove "type" here using utility-types

Reply:
Yep good catch